### DISCUSSION TITLE
Maven and Netbeans Issues in Running Program

### DISCUSSION THREAD

**Discussion by kbitz99** [winder/Universal-G-Code-Sender#1769] [Help]
I'm dealing with issues using Maven to run the ugs program, I am unfamiliar with Maven so any help would be greatly appreciated!
First, when building the ugs core project I get several NullPointer errors 
![image](https://user-images.githubusercontent.com/51766213/147776770-0733892a-53cc-405d-913b-f5bf6c8f16e9.png)
Next, the last report when building ugs-core project it seems to suggest that there is an issue with dependencies.
![image](https://user-images.githubusercontent.com/51766213/147776888-c0469e4c-5693-46c8-981e-9f6f7ee1c030.png)

It is also worth mentioning that I run into similar errors related to dependancies when building ugs-classic or ugs-platform-parent
If anyone has dealt with something similar help would be greatly appreciated!

For more info, I'm running netbeans 9.0, java 17.0.1, Maven 3.8.4

**Comment by breiler** [comment]
Sorry for the late reply. I have no idea why those tests are failing on your machine. What branch are you compiling?
For you getting forward, you could disable the tests bu using `mvn clean install -DskipTests`

Also try to use an older version of Java. I have only successfully been able to run UGS using Java 8 and Java 13 due to some library issues.
